lib/oxide-vpc/src/engine/geneve.rs

```diff
@@ -84,28 +149,24 @@ impl<'a> OptionCast<'a> for ValidOxideOption<'a> {
     }
 }
 
+/// Geneve multicast option body carrying replication strategy information.
+///
+/// This option encodes the replication scope as a 2-bit field in the top two
+/// bits of the first byte of the option body. The remaining 30 bits are
+/// reserved for future use. The replication strategy determines whether the
+/// packet is delivered to local guest instances (External), underlay
+/// infrastructure destinations (Underlay), or both (All).
 #[derive(Debug, Clone, Ingot, Eq, PartialEq)]
 #[ingot(impl_default)]
 pub struct MulticastInfo {
+    /// Replication scope encoded as a u2 (top 2 bits of the first byte).
+    /// Values map to `Replication::{External, Underlay, All, Reserved}`.
```

Comment:
comment nit: I think the representation and enum variants are already shown unambiguously by the code.